python_transport/wirepas_gateway/configure_node.py

```diff
@@ -0,0 +1,339 @@
+# Copyright 2020 Wirepas Ltd licensed under Apache License, Version 2.0
+#
+# See file LICENSE for full license details.
+#
+import binascii
+import argparse
+
+from enum import Enum
+
+from wirepas_gateway.dbus.dbus_client import BusClient
+
+
+class NodeRole:
```

Comment:
What happens if you set an invalid combination (like Sink autorole) does this fail at some point with a clear error message ?